CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## NEXT - TBD
 ### Fixed
-
+- FSDP: fixed an assert for checking if the params of a module requires_grad=True [#761]
```

Comment:
I have an upcoming PR with more details added to the Changelog since I did not have it as part of my PRs. Realized it once the tests were done and did not want to wait another 4 hrs for the tests to pass. Do you want to revert this for now?

Reply:
Sure.